test/mock-login/server.js

```diff
@@ -2,8 +2,8 @@ var Boom = require('boom');
 var Hapi = require('hapi');
 
 module.exports = function() {
-  var server = new Hapi.Server({ debug: false });
-  server.connection({
+  var server = new Hapi.Server({
```

Comment:
`server.connection` is not a thing anymore so we pass the host, port into the constructor.